.esbuild/util.cjs

```diff
@@ -79,8 +81,11 @@ exports.esmCoreBuild = (override) => {
  */
 exports.iifeBuild = (override = { minify: true }) => {
   return buildOptions({
-    entryPoints: getOutFiles(override.minify ? '.min' : ''),
+    entryPoints: getEntryPoints(override.minify ? '.min' : ''),
     format: 'iife',
+    footer: {
+      js: 'mermaid = mermaid.default;',
+    },
```

Comment:
@aloisklink this is a hack I tried out to fix `mermaid.initialize is not a function`. 
We need to figure out the root cause on why it's not emitted properly.

Reply:
Hmmmm, I wonder if it's a TypeScript issue, I vaguely remember having similar issues when converting CommonJS code using `module.exports = example;`, see https://github.com/microsoft/TypeScript/issues/2719

Reply:
Webpack is handling it somehow.